### PR TITLE
process.env scoped to .env file in the root directory of the project

### PR DESCRIPTION
**Pull request by SaiKUndurthi**
making process.env scoped to .env file in the root directory of the project instead of setting up an environment variable. This would avoid the variable from being "undefined"

### CODE REVIEW

File: routes.js [hunk 1]
```diff
@@ -8,6 +8,7 @@
 //
 const fs = require("fs");
 const https = require("https");
+const dotenv = require("dotenv");
```
Comment (greenido) [inline routes.js:11]: Good idea. At Glitch you don't need it but ppl that will install it will need dotenv pkg.

File: routes.js [hunk 2]
```diff
@@ -103,6 +104,11 @@ var routes = function(app) {
  */
 function sendSlackMessage(messageBody) {
   try {
+
+    // Loads environment variables from a .env file into process.env
+    // console.log("------------------------"+process.env.SLACK_WEBHOOK_URL);
+    dotenv.config();
```
Comment (greenido) [inline routes.js]: We should put it at the beginning of the file so the developer will get the acknowledgement when the script load and not only on sending msgs.

File: README.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ So you will have all the modules that we are using.
 1. Defined Xray's webook by following [these steps](https://www.jfrog.com/confluence/display/JFROG/Configuring+Xray#ConfiguringXray-ConfiguringWebhooks).
    If you wish to see (even) better tutorial check this [post](https://greenido.wordpress.com/?p=9820).
 
-1. Copy this [project](https://github.com/greenido/jfrog-xray-2-slack-example) and change (in .env file):
+1. Copy this [project](https://github.com/greenido/jfrog-xray-2-slack-example) and change (in .env file placed in the root directory):
```
Comment (greenido) [inline README.md:43]: good idea - maybe, we should add .env file there but just empty.

**Review by greenido — COMMENTED** [review]
Good ideas... please see my comments.

**Comment by SaiKUndurthi** [comment]
minor adjustments:
- Added an empty .env file
- Called dotenv.config() at the start

**Pull request merged**